packages/browser/src/Component/Widget/Server/SPARQL/View/SparqlQueryWindow.js

```diff
@@ -19,8 +21,16 @@ export class SparqlQueryWindow extends Window {
    * @param {SparqlEndpointResponseProvider} sparqlProvider The SPARQL Endpoint Response Provider
    * @param {CityObjectProvider} cityObjectProvider The City Object Provider
    * @param {LayerManager} layerManager The UD-Viz LayerManager.
+   * @param {object} configSparql The sparqlModule view configuration.
+   * @param {object} configSparql.queries Query configurations
+   * @param {object} configSparql.queries.title The query title
+   * @param {object} configSparql.queries.filepath The path to the file which contains the query text
+   * @param {object} configSparql.queries.formats Configuration for which visualizations are allowed
+   *                                              with this query. Should be an object of key, value
+   *                                              pairs. The keys of these pairs should correspond
+   *                                              with the cases in the updateDataView() function.
    */
-  constructor(sparqlProvider, cityObjectProvider, layerManager) {
+  constructor(sparqlProvider, cityObjectProvider, layerManager, configSparql) {
```

Comment:
rename configSparqlWidget or configSparqlWidgetQueries and then just pass he actual configSparql.queries object